__tests__/buildTranslationFiles.spec.ts

```diff
@@ -224,6 +236,28 @@ describe('buildTranslationFiles', () => {
         assertResult(type, expected.admin, 'admin-page/');
         assertResult(type, expected.nested, 'nested/scope/');
       });
+
+      it('should work with selectTranslate key arrays', () => {
```

Comment:
I set them up so they would specifically test only the key arrays. Is it not preferable to have more specific test cases instead of the first general 'should work with service' test?

Reply:
I do agree with what you are saying generally speaking, but since there are a lot of permutations creating a specific `it` for each might be a lot.
Also, the current permutations are all tested within the same files and not separated into test cases (especially in the template tests), this might be a change we will do further down the road 🙂 

For now, please unify the 2 `it`s to one: `it('should work when passing an array of keys')`, there is no difference between the methods.

Reply:
If I also revert the assertResult change to add `toMatchObject`, then this new unified test case will be the same as the first `it should work with service`, and won't provide any value besides the name.

Reply:
I understand we need the functionality added, but I think the solution should be different 🙂 

* Rename `assertResult` to `assertTranslation` and create a new function called `assertPartialTranslation`.
* Convert the parameters to an options object instead of individual params (`assertTranslation({type , path, expected})` with the proper typing.
* Create a new function called `loadTranslationFile`.

Reply:
Do you run `npm run commit` for every update to the PR?

Reply:
At the end of all the fixes, squash all your commits into one and force push

Reply:
applied the updates you requested.